conformance/lang/expressions/shift-expression/right_shift.balt

```diff
@@ -0,0 +1,4537 @@
+    Ints _ = x >> a; // @error expected 'Ints' but found 'int'
+}
+
+Test-Case: error
+Description: Test resultant type of the '>>' expression with 'int:Signed8' as the LHS operand and optional 'int' as
+             the RHS operand.
+Labels: int, int:Signed8, optional-type, shift-expr, unary-minus, var
+
+function errorFunction(int:Signed8 x, int? a) {
+    var x1 = x >> a;
+    int:Signed8 _ = x1; // @error  expected 'int:Signed8' but found 'int?'
+    var x2 = x >> a;
+    int _ = x2; // @error expected 'int' but found 'int?'
+}
+
+Test-Case: error
+Description: Test resultant type of the '<<' expression with 'int:Signed8' as the LHS operand and optional builtin
+             subtypes of integer as the RHS operand.
+Labels: decimal, float, int:Signed16, int:Signed32, int:Signed8, int:Unsigned16, int:Unsigned32, int:Unsigned8,
+        shift-expr, optional-type
+
+function errorFunction(int:Signed8 a, int:Signed8? b, int:Unsigned8? c, int:Signed16? d, int:Unsigned16? e, int:Signed32? f,
+                        int:Unsigned32? g) {
+    int:Signed16 _ = a >> b; // @error expected 'int:Signed16' but found 'int?'
+    float _ = a >> c; // @error expected 'float' but found 'int?'
+    int:Unsigned16 _ = x3; // @error expected 'int:Unsigned16' but found 'int?'
+    int:Signed16 _ = a >> e; // @error expected 'int:Signed16' but found 'int?'
+    decimal _ = a >> f; // @error expected 'decimal' but found 'int?'
+    int:Unsigned16 _ = a >> g; // @error expected 'int:Unsigned16' but found 'int?'
+}
+
+Test-Case: error
+Description: Test resultant type of the '>>' expression with 'int:Signed8' as the LHS operand and optional byte as
+             the RHS operand.
+Labels: byte, int, int:Signed8, optional-type, shift-expr
+
+function errorFunction(int:Signed8 x, byte? a) {
+    int:Signed8 _ = x >> a; // @error  expected 'int:Signed8' but found 'int?'
+    int _ = x >> a; // @error expected 'int' but found 'int?'
+}
+
+Test-Case: output
+Description: Test '>>' expression with 'int:Signed8' operands in function call and method call.
+Labels: function-call-expr, function-defn, int:Signed8, method-call-expr, shift-expr, unary-minus, value:toBalString,
+        value:toString
+
+function init() {
+    int:Signed8 a = -122;
+    int:Signed8 b = -123;
+    int:Signed8 c = 12;
+    int:Signed8 d = 2;
+
+    leftShiftExpr(a >> b); // @output -4
+    leftShiftExpr(a >> d); // @output -31
+    leftShiftExpr(c >> d); // @output 3
+    leftShiftExpr(c >> b); // @output 0
+
+    io:println((a >> b).toString()); // @output -4
+    io:println((a >> d).toString()); // @output -31
+    io:println((c >> d).toString()); // @output 3
+    io:println((c >> b).toString()); // @output 0
+
+    io:println((a >> b).toBalString()); // @output -4
+    io:println((a >> d).toBalString()); // @output -31
+    io:println((c >> d).toBalString()); // @output 3
+    io:println((c >> b).toBalString()); // @output 0
+}
+
+function leftShiftExpr(int x) {
```

Comment:
Need to change the function name to rightShiftExpr

Reply:
My bad. Will update. Thanks!.